tsinfer/inference.py

```diff
@@ -490,6 +607,111 @@ def match_samples(
     return ts
 
 
+def map_additional_mutations(
+    sample_data,
+    tables,
+    *,
+    sample_id_map=None,
+    sites_exist=True,
+    check_consistency=True,
+    progress_monitor=False,
```

Comment:
This is a nasty way to pass the tsinfer progress monitor to a more general function. What's the proper way to do this, @jeromekelleher ?

Reply:
Good point! Done